src/gennylib/include/gennylib/context.hpp

```diff
@@ -226,10 +227,11 @@ class PhaseContext;
  */
 class ActorContext final : public V1::ConfigNode {
 public:
-    ActorContext(YAML::Node node, WorkloadContext& workloadContext)
+    ActorContext(YAML::Node node, WorkloadContext& workloadContext, bool hasApmOpts)
```

Comment:
please document this and/or choose a better name for the param `apm` means nothing to me 😄